compose/cli/main.py

```diff
@@ -1602,3 +1603,8 @@ def warn_for_swarm_mode(client):
             "To deploy your application across the swarm, "
             "use `docker stack deploy`.\n"
         )
+
+
+def set_no_color_if_clicolor(no_color_flag):
+    return True if no_color_flag or ('CLICOLOR' in os.environ and os.environ['CLICOLOR'] == "0") \
```

Comment:
Can we just `return no_color_flag or ('CLICOLOR' in os.environ and os.environ['CLICOLOR'] == "0")` ?

Reply:
Of course, done ;-)